associacao-brasileira-de-normas-tecnicas-eipp.csl

```diff
@@ -0,0 +1,314 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="pt-BR">
+  <info>
+    <title>Grupo de Pesquisa Educação Infantil e Políticas Públicas da UNIRIO - ABNT (Portuguese - Brazil)</title>
+    <title-short>ABNT-EIPP</title-short>
+    <id>http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-eipp</id>
+    <link href="http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-eipp" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
+    <link href="http://www.tovar.com.br/citationstyles/" rel="documentation"/>
```

Comment:
Does the "Grupo de Pesquisa Educação Infantil e Políticas Públicas da UNIRIO" have a website? Do they have an online style guide that describes this style format?

(I also noticed that http://www.tovar.com.br/citationstyles/ has a link to https://www.zotero.org/support/dev/citation_styles/csl_0.8.1_syntax, which is very outdated. The current version of CSL, 1.0.1, is described at http://docs.citationstyles.org/en/stable/specification.html)

Reply:
Good morning! Thanks for the reply! There are a lot of people from Brazil universities waiting to use our style!

Here are the answers:
1) We are still working on a new version for our website. It’s address is www.tovar.com.br/eipp but you would not find nothing there until december 10th.

2) Our style guide that describe our style format is located here: http://www.tovar.com.br/citationstyles/index.html but when the new website launches we will have a pretty much detailed document.

By the way, that for the link I will update it right away.

Regards!

Reply:
Can you tell us a little bit more about the what the style will be used for exactly?

It looks like UNIRIO is the Universidade Federal do Estado do Rio de Janeiro. So the "Grupo de Pesquisa Educação Infantil e Políticas Públicas" is a research group within the university? Who is the target audience for the style? People who write reports for this group? Or does the "Grupo de Pesquisa Educação Infantil e Políticas Públicas" have its own journal?

Reply:
Yes! Of course!

The "Grupo de Pesquisa Educação Infantil e Políticas Públicas" (EIPP) is a graduate group (Master's and Doctoral Degree) from UNIRIO that develops researches that analyze the proposals, implementations and monitor public policies for Early Childhood Education in the municipalities of the state from Rio de Janeiro. Our group is formally registered in the Directory of Research Groups in Brazil (http://dgp.cnpq.br/dgp/espelhogrupo/6485853444373693) coordinated by CNPq (http://www.cnpq.br/web/guest/apresentacao_institucional /) which is an agency of the Ministry of Education of Brazil.

We are researched from the educational area, we do not have our own, journal, newspaper or magazine, but we publish intensely in Brazilian journals specializing in education that require the ABNT standard with certain formatting peculiarities.

The intention of developing a style is to help other research groups, especially those of the National Association of Postgraduate and Research in Education (http://www.anped.org.br/sobre-anped), begin to use tools such as Mendeley and Zotero (both little known in our area), as well as to simplify the researcher's work by formatting his articles, books and texts in general.

I am completely willing to give you more information and if the fact that the sites are in Portuguese causes you any discomfort, I can try to translate some of them for you to English.

Reply:
Thanks for the clarification. The main reason I'm asking is that we always try to link each of our CSL styles to an officially published citation style guide. See https://github.com/citation-style-language/styles/wiki/Criteria-for-Accepting-Styles for a bit more information.

> we publish intensely in Brazilian journals specializing in education that require the ABNT standard with certain formatting peculiarities.

Is publishing in these journals then the main use of this style? If these journals clearly spell out their citation style requirements, it might be better to add this citation style under the names of these journals. (we have so-called "dependent" CSL styles that make it simple for multiple CSL styles to share the exact same citation format, in case all these journals use the same reference format)

> if the fact that the sites are in Portuguese causes you any discomfort, I can try to translate some of them for you to English.

No, don't bother. Google Translate is quite excellent nowadays.

Reply:
Perfect! Should I do anything else or just wait?

Reply:
Well, my question is whether it would make sense to, instead of naming the style after "Grupo de Pesquisa Educação Infantil e Políticas Públicas da UNIRIO", to create styles for the journals the group is publishing in instead.

Reply:
oh! I am sorry... Thanks for your suggestions but I don't think so. These standards are not specific for each journal. They are a common sense of the educational area. In other words, we need to follow the ABNT but this standard sometimes give us options like "you can use bold or italic" for the title or "you can use the complete name or the short name" for the author. What we establish in our style code was the preferable options for the majority of the researches.

Reply:
Okay, thanks. And apart from your own work at http://www.tovar.com.br/eipp, nobody has posted a detailed description of this "common sense" version yet?

Reply:
No, there is nothing documented.

Reply:
Would a title like `<title>Universidade Federal do Estado do Rio de Janeiro - Educação Infantil e Políticas Públicas - ABNT (Portuguese - Brazil)</title>` work for you? We always put the main organization in front, and try to avoid abbreviations.

Reply:
If it fits better... No problem at all!